### ISSUE TITLE
 Responsiveness for Product page 

### ISSUE THREAD

**Issue by NelisVanWijk** [teamforus/forus#855]
##mobile problem

Images on products page don't scale well especially when the product itself has a long name. 
maybe don't show images when on mobile site or put them somewhere else. 
![afbeelding](https://user-images.githubusercontent.com/25053173/55549031-35137080-56d5-11e9-9ddd-397c0e975031.png)

**Comment by maxvisser** [comment]
we were also thinking about shortening products titles after 25 characters with dots example:
Een hele mooie batavus fiets - Maat xxl - vrouw...

**Comment by NelisVanWijk** [comment]
Yes that sounds good, and maybe it's a bit unnecessary to show the title of the product twice

**Comment by NelisVanWijk** [comment]
![afbeelding](https://user-images.githubusercontent.com/25053173/55552375-7a3ba080-56dd-11e9-99a8-e63e4143110f.png)

as addition to this issue the images of the list of products don't scale good either
